### PR TITLE
Updated Read me about category name

### LINKED ISSUE #23: Update README.md for directions on Application Category
I have noticed that most contributors creating their own app, often copy the app.json file from the boilerplate but forget to modify the Application Category, since we are now classifying all the apps into 6 main categories

1. Search
2. Visualizer
3. Game
4. Scraper
5. Internet of Things
6. Accounts

New apps should try to have one of these as their categories, and should declare new application categories only if absolutely necessary.

### PR DESCRIPTION
**Pull request by Marauderer97**
Added documentation to preserve category structure
Fixes #23

### CODE REVIEW

**Comment by mariobehling** [comment]
Please resolve former commit errors first before merging: https://github.com/fossasia/apps.loklak.org/pull/22

Thank you.

File: README.md [hunk 1]
```diff
@@ -76,6 +76,14 @@ Some of the applications in the repository include-
   - Call http://api.loklak.org/api/apps.json to see if your ```app.json``` is included in the app list
   - Check if all declarations in your ```app.json``` relate to your own app
     (if you copy-pasted another ```app.json```, you may have forgotten to change some fields)
+  - Under Application Category try and choose one of the following Categories:
+  a. Accounts
+  b. Game
+  c. Visualizer
```
Comment (daminisatya) [inline README.md]: Can I know who provided these categories?

File: README.md [hunk 2]
```diff
@@ -76,6 +76,14 @@ Some of the applications in the repository include-
...
+  d. Scraper
+  e. Search
+  f. Internet of Things
+  - List an appropriate Application Sub Category too
```
Comment (daminisatya) [inline README.md]: As Mario mentioned, please provide a link to your local setup where we can review.

**Comment by daminisatya** [comment]
@Marauderer97 any update?

**Comment by Marauderer97** [comment]
@daminisatya Please see issue #12 or pull request #22

**Comment by Abhi2424shek** [comment]
Now whats the status then

**Pull request merged**